lib/lotus/routing/parsing/json_parser.rb

```diff
@@ -3,6 +3,13 @@
 module Lotus
   module Routing
     module Parsing
+      # Json parsing error
+      # This is raised when the json parser fails to parse a json string.
+      #
+      # @since x.x.x
+      class JsonParsingException < ::StandardError
```

Comment:
This should be `BodyParsingError`.

Third party parsers may want to raise the same exception. For instance:

```ruby
class XmlParser
  # ...

  def parse(body)
    # parse xml
    rescue SomeXmlParsingException => e
      raise Lotus::Routing::Parsing::BodyParsingError.new(e)
  end
end
```

We can have only one exception for all the MIME type/parsers. From a final developer perspective, it's easier to catch only one exception:

```ruby
begin
  router.call(env)
rescue Lotus::Routing::Parsing::BodyParsingError => e
  # handle exceptions for JSON/XML/whatever ..
end
```